lib/bios/menus.robot

```diff
@@ -0,0 +1,77 @@
+*** Settings ***
```

Comment:
@SergiiDmytruk @Mixss   Is it in sync with https://github.com/Dasharo/open-source-firmware-validation/pull/25/files or you independently came up with the same file name?

Reply:
Not in sync.  I deliberately used the same filename as contents is similar and this way conflicts will be obvious.

Reply:
Not sure where I should start reviewing then.